File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h = 0;
    int x = 1;
    int check = 1;
    int y = 1;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (x = 0; x < h ; ++x)
    {
        for (int i = 1; i < h - x; i++)
        {
            printf(" ");
        }
        for (int j = 0; j <= x; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50

h = -1

while True:
    h = cs50.get_int("Height: ")
    if 0<h<9:
        break


for y in range(0, h):
    for x in range(0, h - y - 1):
        print(" ", end="")
    for x2 in range(0, y + 1):
        print("#", end="")
    print("  ", end="")
    for x3 in range(0, y + 1):
        print("#", end="")
    print("")
